doc/classes/Vector3.xml

```diff
@@ -309,6 +309,14 @@
 				Returns the result of reflecting the vector from a plane defined by the given normal [param n].
 			</description>
 		</method>
+		<method name="rotate_toward">
+			<return type="Vector3" />
+			<param index="0" name="to" type="Vector3" />
+			<param index="1" name="rotation_delta" type="float" />
+			<param index="2" name="length_delta" type="float" />
+			<description>
```

Comment:
```suggestion
			<description>
				Returns the result of rotating this vector towards [param to], by increment [param delta] (in radians).
				Passing a negative [param delta] will rotate toward the opposite of [param to].
```
Writing docs for the vectors is tricker than for the previous two methods.
Changing inverse to opposite as to avoid confusion with Vector3.Inverse()